plugin.rb

```diff
@@ -84,6 +84,26 @@ class Engine < ::Rails::Engine
     end
   end
 
+  # DISCOURSE CALENDAR
+
+  %w[
+    ../app/controllers/admin/admin_discourse_calendar_controller.rb
+    ../app/controllers/admin/discourse_calendar/admin_holiday_regions_controller.rb
+    ../app/controllers/admin/discourse_calendar/admin_holidays_controller.rb
+  ].each { |path| load File.expand_path(path, __FILE__) }
+
+  ::ActionController::Base.prepend_view_path File.expand_path(
```

Comment:
We don't need to prepend the views given that there aren't any yet. And since this is an admin endpoint, I suspect there won't be any Rails views, the Ember frontend will handle this.

Reply:
Oh yes, good catch, thank you! Totally makes sense! :facepalm: :joy: :+1: